CARET_trace/src/ros_trace_points.cpp

```diff
@@ -85,6 +172,8 @@ void run_caret_trace_node()
   auto trace_node = std::make_shared<TraceNode>(node_name_base, option, lttng, data_container);
   RCLCPP_INFO(trace_node->get_logger(), "%s started", trace_node->get_fully_qualified_name());
 
+  ignore_rcl_timer_init = is_ros2_launch_command();
```

Comment:
This PR is kind of workaround, so please add explanation why we ignore rcl_timer_init in ros2 launch.

```suggestion
  // rcl_timer_init() is called every on second from ComposableNodeContainer, which increases wait time to start CARET record.
  // rcl_timer_init event is ignored if the process is ROS 2 launch.
  // ref: https://github.com/ros2/rclpy/blob/e29a7ec1985ac0113eaf9aca41858d66f19f1215/rclpy/rclpy/executors.py#L705
  ignore_rcl_timer_init = is_ros2_launch_command();
```

Reply:
Updated with comments as I reviewed the conditions